Mathlib/Algebra/Algebra/Hom.lean

```diff
@@ -48,6 +48,12 @@ class AlgHomClass (F : Type*) (R A B : outParam Type*)
 -- 15% if we would do so (see benchmark on PR https://github.com/leanprover-community/mathlib4/pull/18040).
 -- attribute [simp] AlgHomClass.commutes
 
+/-- The algebra morphism underlying `algebraMap` -/
+def Algebra.algHom (R : Type*) [CommSemiring R] (S : Type*) [Semiring S] [Algebra R S] :
```

Comment:
It seems to me this is a particular case of:
```lean
def Algebra.algHom (R A B : Type*)
    [CommSemiring R] [CommSemiring A] [Semiring B] [Algebra R A] [Algebra R B]
    [Algebra A B] [IsScalarTower R A B] :
    A →ₐ[R] B where
  toRingHom := algebraMap A B
  commutes' r := by simpa [Algebra.smul_def] using smul_assoc r (1 : A) (1 : B)
```
(This should be moved before the definition of `AlgHomClass`.)